packages/application-shell/src/components/navbar/navbar.spec.js

```diff
@@ -1,13 +1,14 @@
-import { shallow } from 'enzyme';
 import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
FYI: when we re-implement the navbar, we can drop these tests and write RTL tests.